OrbitCore/OrbitModule.cpp

```diff
@@ -108,45 +107,62 @@ Function* Pdb::FunctionFromName(const std::string& a_Name) {
 }
 
 //-----------------------------------------------------------------------------
-std::string FindSymbols(const std::string& module_path) {
+std::unique_ptr<ElfFile> FindSymbols(std::string& module_path) {
+  auto module_elf_file = ElfFile::Create(module_path);
```

Comment:
I would avoid using auto in cases like this. For example it's not immediately clear that this is a unique_ptr.